src/popup/components/AccountDetailsTransactionsBase.vue

```diff
@@ -25,6 +25,8 @@ import {
 
 import TransactionList from '@/popup/components/TransactionList.vue';
 
+let initialized = false;
```

Comment:
Can't we use the existing `isPageActive` ref instead of `initialized`? (and update it in the correct places)

Reply:
The page will become active before the `IonRouterOutlet` will render it, and this will break the parent element size calculation.